contractcourt/commit_sweep_resolver.go

```diff
@@ -271,10 +237,6 @@ func (c *commitSweepResolver) Encode(w io.Writer) error {
 		return err
 	}
 
-	if c.sweepTx != nil {
```

Comment:
since we are in fact leaving data on-disk that will no longer be read, we should add a comment here explaining that there may be instances where this transaction is still present. this way we don't forget to handle/migrate if we ever need to append data to the on-disk format. would probably be useful for the decode case as well

Reply:
Comments added